manifests/f/FastFetchCLI/FastFetch/2.13.2/FastFetchCLI.FastFetch.locale.en-US.yaml

```
@@ -0,0 +1,23 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: FastFetchCLI.FastFetch

```

```suggestion
PackageIdentifier: Fastfetch-cli.Fastfetch
```
Please update file name and path to match this change.